packages/flutter/lib/src/material/dialog.dart

```diff
@@ -83,7 +92,13 @@ class Dialog extends StatelessWidget {
               elevation: 24.0,
               color: _getColor(context),
               type: MaterialType.card,
-              child: child,
+              child: new Semantics(
```

Comment:
here and elsewhere, why not use the RouteName widget?

Reply:
I didn't make RouteName insert a scopesRoute semantics, since most of the use will be providing names to what users think of as routes.  For Dialogs, Drawers, and PopupMenus we have a named parameter and default semantic values which I will add when I do the localizations - in addition to the inferred semantics from title like widgets